README.md

```diff
@@ -78,16 +78,17 @@ Following inputs can be used as `step.with` keys
-|------------------|----------|------------------------------------|
-| `builder`        | String   | Builder instance (see [setup-buildx](https://github.com/docker/setup-buildx-action) action) |
-| `files`          | List/CSV | List of [bake definition files](https://github.com/docker/buildx/blob/master/docs/reference/buildx_bake.md#file) |
-| `targets`        | List/CSV | List of bake targets |
-| `no-cache`       | Bool     | Do not use cache when building the image (default `false`) |
-| `pull`           | Bool     | Always attempt to pull a newer version of the image (default `false`) |
-| `load`           | Bool     | Load is a shorthand for `--set=*.output=type=docker` (default `false`) |
-| `push`           | Bool     | Push is a shorthand for `--set=*.output=type=registry` (default `false`) |
-| `set`            | List     | List of [targets values to override](https://github.com/docker/buildx/blob/master/docs/reference/buildx_bake.md#set) (eg: `targetpattern.key=value`) |
+| Name       | Type     | Description                                                                                                                                          |
+|------------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `builder`  | String   | Builder instance (see [setup-buildx](https://github.com/docker/setup-buildx-action) action)                                                          |
+| `files`    | List/CSV | List of [bake definition files](https://github.com/docker/buildx/blob/master/docs/reference/buildx_bake.md#file)                                     |
+| `targets`  | List/CSV | List of bake targets                                                                                                                                 |
+| `no-cache` | Bool     | Do not use cache when building the image (default `false`)                                                                                           |
+| `pull`     | Bool     | Always attempt to pull a newer version of the image (default `false`)                                                                                |
+| `load`     | Bool     | Load is a shorthand for `--set=*.output=type=docker` (default `false`)                                                                               |
+| `push`     | Bool     | Push is a shorthand for `--set=*.output=type=registry` (default `false`)                                                                             |
+| `set`      | List     | List of [targets values to override](https://github.com/docker/buildx/blob/master/docs/reference/buildx_bake.md#set) (eg: `targetpattern.key=value`) |
+| `remote`   | String   | Read bake definition files from [remote context](https://github.com/docker/buildx/blob/master/docs/guides/bake/file-definition.md#remote-definition) |
```

Comment:
It seems like multiple remotes are allowed? If so, should this be a `List` instead?

Reply:
Only the first arg is checked as remote atm: https://github.com/docker/buildx/blob/3cf549a7f7ee015cabe99d4a08e4b9aabfc83ccc/commands/bake.go#L43

Reply:
Oh actually you're right we can define remote bake files and remote context with a 2nd arg: https://github.com/docker/buildx/blob/3cf549a7f7ee015cabe99d4a08e4b9aabfc83ccc/commands/bake.go#L46-L50